electron-app/src/components/accounts/ManualBalancesList.vue

```diff
@@ -47,7 +47,13 @@
             ></amount-display>
           </template>
           <template #item.usdValue="{ item }">
-            <amount-display fiat :value="item.usdValue"></amount-display>
+            <amount-display
+              fiat
+              :amount="item.amount"
+              :fiat-currency="item.asset"
```

Comment:
isn't the name confusing though? the asset can be non-fiat too right?

Reply:
I don't think double conversion will ever happen if the asset is not fiat.

Reply:
So you only use it for columns that display fiat currencies right? then it is my misunderstanding.

Reply:
I think so, @LefterisJP maybe you can clarify if you had other instances in mind?

Reply:
This is for the solution to the double conversion: https://github.com/rotki/rotki/issues/957  ?

I am not sure what we look at here, but if this is the manually tracking balances value column (at least at the moment), it can only be displayed in a FIAT currency as the user can only select one of the given FIAT currencies from the top right dropdown menu.

Reply:
@isidorosp do you think it makes sense to merge `fiat` and `fiat-currency` into one property?

Reply:
> @isidorosp do you think it makes sense to merge `fiat` and `fiat-currency` into one property?

Hmm.. Probably! Are you thinking of passing an array / object or just if "fiat-currency" exist then basically assume "fiat===true"?

Reply:
Keep fiat currency as an optional string property that defaults to null. Then do a `!!fiatCurrency` in the template/code. This should evaluate to false if null and true if not.